### PR TITLE
More diffs in translations files

### PR DESCRIPTION
**Pull request by SidVal**
fixes https://github.com/SidVal/docs/issues/35

### CODE REVIEW

File: content/docs/contribute/contribute@es.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 ---
 $title: Contribuciones
+$order: 0
```
Comment (bpaduch) [inline content/docs/contribute/contribute@es.md]: For the record, if the English file has these variables in the front matter, any localized files do not need them - they inherit them.  So this change is unnecessary.

File: content/docs/guides/discovery@es.md [hunk 2]
```diff
@@ -76,8 +85,11 @@ Ejemplo:
 
 Puedes encontrar más ejemplos en la [carpeta de ejemplos de ampproject](https://github.com/ampproject/amphtml/tree/master/examples/metadata-examples), que incluye la sintaxis de atributo HTML alternativa).
 
-Nota: Esta definición de Schema.org es un requisito para que el contenido sea apto para aparecer en la demostración del [carrusel de noticias de la Búsqueda de Google (prueba la versión para móvil)](https://g.co/ampdemo).
-Consulta también las [Historias principales con AMP](https://developers.google.com/structured-data/carousels/top-stories) y la [herramienta de pruebas de datos estructurados](https://developers.google.com/structured-data/testing-tool/).
+{% call callout('Read on', type='read') %} Visita estas páginas para más información sobre datos estructurados:
```
Comment (bpaduch) [inline content/docs/guides/discovery@es.md]: Leer mas

File: content/docs/contribute/contribute@es.md (hunk 1 above)
Comment (SidVal) [inline content/docs/contribute/contribute@es.md]: Ah, OK. I just copy that from source (english file).
I delete the "order" ?

File: content/docs/guides/discovery@es.md (hunk 2 above)
Comment (SidVal) [inline content/docs/guides/discovery@es.md]: My bad. Fixed.

File: content/docs/contribute/contribute@es.md (hunk 1 above)
Comment (bpaduch) [inline content/docs/contribute/contribute@es.md]: Yes, just title is needed.

File: content/docs/guides/discovery@es.md [hunk 3]
```diff
@@ -1,6 +1,11 @@
 ---
 $title: Hacer que una página sea visible
+$order: 1
```
Comment (bpaduch) [inline content/docs/guides/discovery@es.md]: Same here -  just title is needed.

**Pull request merged**